components/desktop/partials/KnitProjectsWidgetItem.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="knit-projects-widget-item">
+    <div class="knit-projects-widget-item__top-section">
+      <img :src="projectLogoSource || 'images/teal-star.png'" alt="" class="knit-projects-widget-item__project-logo">
```

Comment:
I recommend not to use `.png` images which not renders correctly on Retina displays. We should use icon fonts wherever we can.

Reply:
Thanks I will change it :)